pkg/storegateway/series_refs_test.go

```diff
@@ -1211,6 +1212,7 @@ func TestOpenBlockSeriesChunkRefsSetsIterator(t *testing.T) {
 				block.meta.MaxTime,
 				newSafeQueryStats(),
 				log.NewNopLogger(),
+				NewBucketStoreMetrics(prometheus.NewRegistry()),
```

Comment:
Can we assert the expected metric is tracked? Just check the value is > 0. Ping me if turns out to be non trivial to check. We already have some utilities to read a metric value somewhere.

Reply:
I pushed https://github.com/grafana/mimir/pull/3661/commits/31c138181b57bfaa15ef351854774dc05a9d7427, which should address this